modules/sde-submodules/assembly-part-relationship/assembly-part-relationship.md

```diff
@@ -5,17 +5,17 @@
 This module use for AssemblyPartRelationship submodel specification and descriptors. It's contain the codes related to AssemblyPartRelationship to validate, parse and transfer data for DigitalTwins and EDC to create aspect twins and data offer.
 
 ---
-#### Version: 1.1.0
-#### AssemblyPartRelationship Aspect Model URN: urn:bamm:io.catenax.assembly_part_relationship:1.1.0#AssemblyPartRelationship
-#### Semantic Id: urn:bamm:io.catenax.assembly_part_relationship:1.1.0
+#### Version: 1.1.1
+#### AssemblyPartRelationship Aspect Model URN: urn:bamm:io.catenax.assembly_part_relationship:1.1.1#AssemblyPartRelationship
+#### Semantic Id: urn:bamm:io.catenax.assembly_part_relationship:1.1.1
 ---
 
 ### Schema
 
 Please find below links for schema details:
 
-- https://portal.int.demo.catena-x.net/semantichub/urn%3Abamm%3Aio.catenax.assembly_part_relationship%3A1.1.0%23AssemblyPartRelationship
-- blob:https://portal.int.demo.catena-x.net/cf9f1c9f-2d31-4b41-82e3-5285e410f76f
+- https://portal.int.demo.catena-x.net/semantichub/urn%3Abamm%3Aio.catenax.assembly_part_relationship%3A1.1.1%23AssemblyPartRelationship
+- blob:https://portal.int.demo.catena-x.net/83e06ce8-21f8-4e31-9349-7a6c2e38a641
```

Comment:
those things are not accessable and you should NOT host or reference our int env AT ALL!

Either do it like other teams and use something like github pages or do something else.

Reply:
Sure, I will ask team to make changes.

Reply:
Suggested changes  has been made.